2022-10-14 15:10:11,163 DEBUG   MainThread:2297 [wandb_config.py:_load_defaults():111] no defaults not found in config-defaults.yaml
2022-10-14 15:10:11,205 DEBUG   MainThread:2297 [cmd.py:execute():871] Popen(['git', 'cat-file', '--batch-check'], cwd=/home/jupyter/papers/regressGAN, universal_newlines=False, shell=None, istream=<valid stream>)
2022-10-14 15:10:11,220 DEBUG   MainThread:2297 [cmd.py:execute():871] Popen(['git', 'rev-parse', '--show-toplevel'], cwd=/home/jupyter/papers/regressGAN, universal_newlines=False, shell=None, istream=None)
2022-10-14 15:10:11,232 DEBUG   MainThread:2297 [cmd.py:execute():871] Popen(['git', 'status', '--porcelain', '--untracked-files'], cwd=/home/jupyter/papers/regressGAN, universal_newlines=False, shell=None, istream=None)
2022-10-14 15:10:11,267 DEBUG   MainThread:2297 [run_manager.py:__init__():535] Initialized sync for cgan_regression/2z45acbs
2022-10-14 15:10:11,287 INFO    MainThread:2297 [run_manager.py:wrap_existing_process():1133] wrapping existing process 2209
2022-10-14 15:10:11,287 WARNING MainThread:2297 [io_wrap.py:register():104] SIGWINCH handler was not None: <Handlers.SIG_DFL: 0>
2022-10-14 15:10:11,293 DEBUG   MainThread:2297 [connectionpool.py:_new_conn():1007] Starting new HTTPS connection (1): pypi.org:443
2022-10-14 15:10:11,344 DEBUG   MainThread:2297 [connectionpool.py:_make_request():465] https://pypi.org:443 "GET /pypi/wandb/json HTTP/1.1" 200 68569
2022-10-14 15:10:11,384 INFO    MainThread:2297 [run_manager.py:init_run():918] system metrics and metadata threads started
2022-10-14 15:10:11,385 INFO    MainThread:2297 [run_manager.py:wrap_existing_process():1150] informing user process we are ready to proceed
2022-10-14 15:10:11,386 INFO    MainThread:2297 [run_manager.py:_sync_etc():1257] entering loop for messages from user process
2022-10-14 15:10:12,196 INFO    Thread-3  :2297 [run_manager.py:_on_file_modified():682] file/dir modified: /home/jupyter/papers/regressGAN/replication/CGAN/wandb/dryrun-20221014_151009-2z45acbs/config.yaml
2022-10-14 16:11:57,863 INFO    MainThread:2297 [run_manager.py:_sync_etc():1280] received message from user process: {"exitcode": 0}
2022-10-14 16:11:57,863 INFO    MainThread:2297 [run_manager.py:_sync_etc():1366] closing log streams and sending exitcode to W&B
2022-10-14 16:11:57,863 INFO    MainThread:2297 [run_manager.py:shutdown():1057] shutting down system stats and metadata service
metadata service
xx1wq:run_manager.py:_sync_etc():1366] closing log streams and sending exitcode to W&B
2022-10-14 15:31:25,664 INFO    MainThread:12947 [37pxx1wq:run_manager.py:shutdown():1057] shutting down system stats and metadata service
2022-10-14 15:39:01,214 INFO    MainThread:28508 [m8ndkv5a:run_manager.py:_sync_etc():1280] received message from user process: {"exitcode": 0}
2022-10-14 15:39:01,215 INFO    MainThread:28508 [m8ndkv5a:run_manager.py:_sync_etc():1366] closing log streams and sending exitcode to W&B
2022-10-14 15:39:01,216 INFO    MainThread:28508 [m8ndkv5a:run_manager.py:shutdown():1057] shutting down system stats and metadata service
2022-10-14 15:59:24,343 INFO    MainThread:8424 [2g0smeqv:run_manager.py:_sync_etc():1280] received message from user process: {"exitcode": 0}
2022-10-14 15:59:24,343 INFO    MainThread:8424 [2g0smeqv:run_manager.py:_sync_etc():1366] closing log streams and sending exitcode to W&B
2022-10-14 15:59:24,343 INFO    MainThread:8424 [2g0smeqv:run_manager.py:shutdown():1057] shutting down system stats and metadata service
2022-10-14 16:11:57,863 INFO    MainThread:2297 [2z45acbs:run_manager.py:_sync_etc():1280] received message from user process: {"exitcode": 0}
2022-10-14 16:11:57,863 INFO    MainThread:2297 [2z45acbs:run_manager.py:_sync_etc():1366] closing log streams and sending exitcode to W&B
2022-10-14 16:11:57,863 INFO    MainThread:2297 [2z45acbs:run_manager.py:shutdown():1057] shutting down system stats and metadata service
